syntax = "proto3";

message WorkerProto {
  uint32 id = 1;
  string name = 2;
  string phone = 3;
  uint32 salary = 4;
}